src/mimic_joint_plugin.cpp

```diff
@@ -160,7 +164,7 @@ namespace gazebo {
         double a = mimic_joint_->GetAngle(0).Radian();
 #endif
 
-        if (abs(angle - a) >= sensitiveness_) {
+        if (fabs(angle - a) >= sensitiveness_) {
```

Comment:
Just out of curiosity. Did this produce any errors or unexpected behavior?

Reply:
Yes, it was giving me always 0 (because of the integer conversion).

Reply:
> Yes, it was giving me always 0 (because of the integer conversion).

Weird. It should fall back to `std::abs` that should have the correct behavior. Anyway..